ddtrace/tracer/log.go

```diff
@@ -36,6 +36,7 @@ type startupInfo struct {
 	Debug                       bool              `json:"debug"`                          // Whether debug mode is enabled
 	AnalyticsEnabled            bool              `json:"analytics_enabled"`              // True if there is a global analytics rate set
 	SampleRate                  string            `json:"sample_rate"`                    // The default sampling rate for the rules sampler
+	SampleRateLimit             string            `json:"sample_rate_limit"`              // The rate limit configured with the rules sampler
```

Comment:
Frankly these two would work better as `SamplingRate` or `SampleRate` and `SamplerRateLimit`. As it is now they are too similar and it takes a while to figure out what they are, even for me who is familiar with the implementation details.

Just a thought...

Reply:
I agree, but I wanted to keep them similar to the names of the ENV variables used to set the values, otherwise it gets further confusing.

Currently:
```
DD_TRACE_SAMPLE_RATE -> SampleRate
DD_TRACE_RATE_LIMIT -> SampleRateLimit
```

I added `Sample` to the front of `RateLimit` because I thought a field just called `RateLimit` was too vague (I think the same is true about the environment variable, but I don't control that). 

This is further unfortunate since `DD_TRACE_SAMPLE_RATE` doesn't *really* set a rate, but a percentage of traces that get sampled.

I don't see much improvement between `SampleRateLimit` and `SamplerRateLimit`, but am not opposed to making that change. If we're ok with `SampleRate` and `SamplerRateLimit` that's fine with me.

If we have a better idea for names for these, that would be even better. The current naming isn't especially good, but I've not been able to come up with something better.
Let me know what you think.